python/py-pint/Portfile

```diff
@@ -4,7 +4,8 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        hgrecco pint 0.8.1
+github.setup        hgrecco pint 0.9
+distname            ${github.version}
```

Comment:
The `distname` should already be set to this by the GitHub portgroup. So this line is redundant and should be removed.